tests/recipes-v2/xtensor/recipe.yaml

```diff
@@ -12,28 +16,42 @@ source:
 
 build:
   number: 0
+  run_exports:
+    - pin_subpackage:
+        name: xtensor
+        max_pin: 'x.x.x'
 
 requirements:
   build:
-    - '{{ compiler("cxx") }}'
+    - compiler: cxx
     - cmake
-    - sel(unix): make
+    - if: unix
+      put: make
```

Comment:
Just checking - `if/put` will be the new way to do it in the next `boa` release?

I'm happy to keep up with the changes and don't want backward compatibility concerns to slow down development - at this stage I think it's more important to experiment to find the right api.

Reply:
Hey, yeah, I am undecided. We're having some discussions with the conda community about this and I wanted to see how it feels. I think we should also have a shorthand `- if(unix): make` that we can easily normalize to the longer form.

This more verbose form adds a lot of "noise" to the recipes...

Reply:
Would love to hear your thoughts. Are you on the conda slack? There is a spec discussion room that recently saw some activity!

Reply:
I'll check out the slack :+1:

The `put` did seem a bit verbose, but not too terrible!